extensions/Worker.Extensions.Tables/src/Worker.Extensions.Tables.csproj

```diff
@@ -14,6 +14,18 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\Worker.Extensions.Abstractions\src\Worker.Extensions.Abstractions.csproj" />
+    <ProjectReference Include="..\..\..\src\DotNetWorker.Core\DotNetWorker.Core.csproj" />
```

Comment:
Need to update the version to match the WebJobs one, it should be `1.2.0-preview1`

    <VersionPrefix>1.2.0</VersionPrefix>
    <VersionSuffix>-preview1</VersionSuffix>